.github/workflows/cd.yml

```diff
@@ -66,6 +66,7 @@ jobs:
           libtorrent==${{ matrix.libtorrent }}
           pyinstaller
           pygame
+          https://github.com/pydron/ifaddr/archive/37cb5334f392f12811d38d90ec891746e3247c76.zip
```

Comment:
This can be put in the requirements.txt with `sys_platform == 'win32'` to only install for Windows